src/components/enterprise-redirects/tests/EnterpriseLearnerFirstVisitRedirect.test.jsx

```diff
@@ -19,14 +21,59 @@ jest.mock('react-router-dom', () => ({
   }),
 }));
 
+const defaultUserSubsidyState = {
+  redeemableLearnerCreditPolicies: [{
+    learnerContentAssignments: {
+      state: 'allocated',
+    },
+  },
+  {
+    learnerContentAssignments: {
+      state: 'accepted',
+    },
```

Comment:
nit: should we include assignment(s) with `state='errored'` and `state='cancelled'` as well?

Reply:
Deferred on the errored state for now since the repo doesn't handle the error state at the moment, but the following PR with the optimizations will handle the error state. Will leave comment open to reference. 👍🏽